<launch>

	<rosparam command="load" file="$(find OryxManager)/params/Front_Left_Wheel.yaml" ns="motors/Drive_Motors/Front_Left_Wheel" />
	<rosparam command="load" file="$(find OryxManager)/params/Front_Right_Wheel.yaml" ns="motors/Drive_Motors/Front_Right_Wheel" />
	<rosparam command="load" file="$(find OryxManager)/params/Back_Right_Wheel.yaml" ns="motors/Drive_Motors/Back_Right_Wheel" />
	<rosparam command="load" file="$(find OryxManager)/params/Back_Left_Wheel.yaml" ns="motors/Drive_Motors/Back_Left_Wheel" />

	<node name="Name" pkg="EposManager" type="EposManager" args="Front_Left_Wheel Front_Right_Wheel Back_Right_Wheel Back_Left_Wheel" ns="motors/Drive_Motors"  output="screen" respawn="true" >
		<param name="Port" value="USB0" />
        <param name="Protocol" value="USB" />
		<param name="Publish_Rate" value="1" />
		
	</node>
		

</launch>
